<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PibotoLt'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;전원&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnOn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>89</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>on</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/power-switch.png</normaloff>../../../../Downloads/power-switch.png</iconset>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PibotoLt'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;내 프리셋&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnRed">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/circle.png</normaloff>../../../../Downloads/circle.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnBlue">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/record.png</normaloff>../../../../Downloads/record.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnGreen">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/button.png</normaloff>../../../../Downloads/button.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LED CONTROL</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BtnOn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>BtnOn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnRed</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>BtnRed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnBlue</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>BtnBlue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnGreen</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>BtnGreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BtnOn()</slot>
  <slot>BtnRed()</slot>
  <slot>BtnBlue()</slot>
  <slot>BtnGreen()</slot>
 </slots>
</ui>
